README.md

```diff
@@ -3,6 +3,10 @@
 The PariQL I.R. Generator (PIG) is a [nanopass](http://nanopass.org/) inspired type domain code generator currently 
 targeting Kotlin.
 
+## API Status
+
+PIG is usable now but its API and the API of the generated Kotlin code should be considered unstable.
```

Comment:
nit: use "under development and might change" VS "unstable"

Reply:
a valid point.  updated.